go-controller/pkg/ovn/ovn.go

```diff
@@ -204,9 +201,10 @@ type Controller struct {
 }
 
 type retryEntry struct {
-	pod        *kapi.Pod
-	timeStamp  time.Time
-	backoffSec time.Duration
+	pod                   *kapi.Pod
+	logicalPortCacheEntry *lpInfo
```

Comment:
add a comment here or rename the variable to indicate this is only used for tracking a delete action